### DISCUSSION TITLE
Embed custom HTML

### DISCUSSION THREAD

**Discussion by benwoodward** [roam-garden/i#51] [Feature requests & Ideas] [answered]
I'd like to be able to create a block in Roam that contains some custom HTML, e.g.:

```html
<form action="https://convertkit.com/signup/new">
<h2>Join Newsletter</h2>
<input>
</form>
```

And then be able to embed that block in a page and for roam.garden to render the HTML that's stored in the block. This would allow me to add newsletter sign up forms in specific places.

**Comment by Stvad** [comment, marked as answer]
I believe this should work right now. Generally valid HTML is rendered & embedding iframe/video tags works. Can you give it a go? 🙂

**Reply by Stvad** [reply]
ok, as mentioned this should work, resolving this for the moment. Please comment if you have further questions/follow up
